scripts/mkcloud

```diff
@@ -1439,7 +1439,8 @@ for cmd in `echo $steplist` ; do
     # this is currently used for the cct function
     cmd_parameters="${cmd#*+}"
     cmd=${cmd%%+*}
-    $cmd "${cmd_parameters//+/ }"
+    TIMEFORMAT="mkcloud step '$cmd' took real=%R user=%U system=%S"
```

Comment:
Then it would need a ';' in the middle to not call /usr/bin/time that does not understand TIMEFORMAT and bash functions.
So better leave it like this.